src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -295,6 +295,7 @@ public struct TypeDescriptor
         {
             public uint FieldCount;
             public CORINFO_FIELD_STRUCT_** Fields; // array of CORINFO_FIELD_STRUCT_*
+            public uint IsExplicitLayout;
```

Comment:
We should be mindful not to waste too much space here, since it is cached (this is 32 bits for a boolean). It's fine as-is right now, but if we add more booleans like this, we would make this `uint` a flags value.